data.js

```diff
@@ -0,0 +1,70 @@
+var CarLot = (function(carlot) {
+  // Defining private varible to store parsed JSON data
+  var data = {}
+  // Function to get JSON data and then call a function to build the HTML page
+  carlot.loadInventory = (function() {
+    var xhr = new XMLHttpRequest();
+    xhr.addEventListener("load", function() {
+        data = JSON.parse(xhr.responseText).cars;
+        carlot.buildHTML(data)
+        carlot.activateEvents()
+    })
+    xhr.open("GET", "inventory.json")
+    xhr.send()
+  })()
+  // Adding a property to the global variable that will return inventory data
+  carlot.getInventory = function() {
+    return data;
+  }
+  // Function to build the HTML page with JSON data returned via loadInventory
+  carlot.buildHTML = function(data) {
+    for (i in data) {
+      if (i % 3 == 0) {
```

Comment:
Nice solution here! This is the first time I've seen anyone wait til i%3==0 to print all three cards. Usually I see them printing one at a time.